core/utils_test.py

```diff
@@ -779,6 +779,14 @@ def test_get_time_in_millisecs_with_complicated_time(self) -> None:
         self.assertEqual(
             dt, datetime.datetime.fromtimestamp(msecs / 1000.0))
 
+    def test_convert_millisecs_time_to_datetime_object(self) -> None:
+        msecs = 1000000.101
+        dt = utils.convert_millisecs_time_to_datetime_object(msecs)
+        self.assertEqual(
+            msecs, (time.mktime(dt.timetuple())) * 1000.0 + (
```

Comment:
The expected value should be hardcoded -- don't include logic in tests.

Reply:
Done